lib/foodcritic/rules.rb

```diff
@@ -26,16 +26,7 @@
   end
 end
 
-rule "FC003",
-     "Check whether you are running with chef server before using"\
-     " server-specific features" do
-  tags %w{portability solo}
-  recipe do |ast, filename|
-    unless checks_for_chef_solo?(ast) || chef_solo_search_supported?(filename)
-      searches(ast)
-    end
-  end
-end
+# FC003 was yanked and the number should not be reused
```

Comment:
# FC003 was a check for Chef server specific features that 
# did not interoperate with the implementation of chef-solo.
# It has been deprecated, and should not be reused.